docs/storage.md

```diff
@@ -307,3 +308,20 @@ config:
 ```
 
 Set the flags `--objstore.config-file` to reference to the configuration file.
+
+##  AliYun OSS Configuration
+Using AliYun OSS object storage, you should first create a bucket , with ACL `Private` & Storage Class `Standard` (I do not test each of `IA` and `Archive` Storage Class ever. if it is not a best choice, please feel free to tell me, I will test it ,for now, it is my best choice). Secondary , before you access to aliyun oss, you should apply for the AccessKey . Go over [https://www.alibabacloud.com/product/oss](https://www.alibabacloud.com/product/oss) for more detail information.
+
+To configure AliYun OSS objstore need a yaml format configuration file, you should specify the parameters list below.
+
+[embedmd]:# (flags/config_oss.txt $)
+```$
+type: OSS
+config:
+  bucket: ""
+  endpoint: ""
+  secret_key: ""
```

Comment:
There is a little confusion here. Users will get used to **access_key and secret_key** or **access_key_id and access_key_secret** just like go sdk does.

Reply:
OK